integration/test/basic/basic_test.go

```diff
@@ -69,7 +74,8 @@ func TestChartLifecycle(t *testing.T) {
 
 	// Test Update
 	l.Log("level", "debug", "message", fmt.Sprintf("updating %s", cr))
-	err = updateChartOperatorResource(helmClient, cr)
+	chartConfigValues.Channel = "5-6-beta"
```

Comment:
What is with this magic string? We have some config above as well which I do not comprehend. Can we at least have some comments so reasoning about it is more easy?